reports-scheduler/config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0"?>
```

Comment:
Should this go below the copyright notice?

Reply:
XML requires the header to be on the top.